qa/integration/services/logstash_service.rb

```diff
@@ -155,6 +155,15 @@ def build_child_process(*args)
       args << feature_config_dir
       puts "Found feature flag. Starting LS using --path.settings #{feature_config_dir}"
     end
+    if java.lang.System.getProperty("java.vm.specification.version").to_i >= 15
+      jvm_option_path = File.join("#{@logstash_home}", "config", "jvm.options")
```

Comment:
know this is still a DRAFT and you're probably only doing this to verify tests :green_circle: 
but having a separate **jvm.options** file based on Java version looks a bit :wilted_flower: 
have been putting in some thinking into this (to avoid CMS) and the best way seems to be what ES is doing already:
`8-13:-XX:+UseConcMarkSweepGC` having a special prefix for conditional opts.

also not sure G1 (default) is the best way forward for LS but that's a separate discussion.

Reply:
This is a nice a suggestion, in first iteration I don't implement the merging of different option files feature that Elasticsearch has, but keep only the optional prefix parsing of options files

Reply:
sounds like a plan. sorry, just noticed that there's been another early review stating this :male_detective: :gun: